doc/3/controllers/document/index.md

```diff
@@ -0,0 +1,55 @@
+---
+code: true
+type: page
+title: create
+description: document:create
```

Comment:
Description are used by Algolia but also by search engine to index pages
```suggestion
Creates a new document
```

Reply:
@Aschen do you mean
```
description: Creates a new document
```

Reply:
Yes, sorry